spec/models/game_spec.rb

```diff
@@ -1,6 +1,12 @@
 require 'rails_helper'
 
 RSpec.describe Game, type: :model do
+  describe "Validations" do
+    it { should have_secure_token(:game_key) }
+    it { should have_secure_token(:intel_key) }
+    it { should have_secure_token(:invite_code) }
```

Comment:
How does the game key and invite code differ? Could they be the same?

Reply:
My thought process is that if the invite code is the same as the key to connect to the game channel, anyone with that key could subscribe to the channel and get game data updates even if they're not actually part of the game. This was a late-night decision though, so there may be alternative ways of handling this I haven't considered. I'd be happy to talk it over first thing when we group up today!